cps/db.py

```diff
@@ -29,7 +29,7 @@
 from sqlite3 import OperationalError as sqliteOperationalError
 from sqlalchemy import create_engine
 from sqlalchemy import Table, Column, ForeignKey, CheckConstraint
-from sqlalchemy import String, Integer, Boolean, TIMESTAMP, Float
+from sqlalchemy import String, Integer, Boolean, TIMESTAMP, Float, or_
```

Comment:
What's the trailing underscore on the end of `or_` all about — can you explain the purpose for the rest of us @deldesir ?  (Thanks!)

Reply:
It was accidentally moved from a previous code and was a redundant import (already imported in https://github.com/iiab/calibre-web/blob/dcb0f14ed22ea4f78767d6cd67ddd19e8a19561e/cps/db.py#L43). It's in fact used to create logical OR expressions between multiple conditions. It allows you to combine conditions and retrieve results that satisfy any one or more of them. Example of its use is in https://github.com/iiab/calibre-web/blob/dcb0f14ed22ea4f78767d6cd67ddd19e8a19561e/cps/db.py#L955